src/main/java/com/kodilla/ecommercee/domain/ProductDto.java

```diff
@@ -0,0 +1,18 @@
+package com.kodilla.ecommercee.domain;
+
+import javax.persistence.Id;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
```

Comment:
Pewnie przyda się na potem też NoArgsConstructor

Reply:
Jak się przyda to dodamy, dla mnie to martwy kod